using-mobile-extensions/mobile-core/signal/README.md

```diff
@@ -0,0 +1,211 @@
+@Override
+public void onCreate() {
+super.onCreate();
+     MobileCore.setApplication(this);
+     try {
+         ...........................
+         Signal.registerExtension();
+         ...........................
+     } catch (Exception e) {
+         //Log the exception
+     }
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+Register the Signal extension in your app's `didFinishLaunchingWithOptions` function:
+
+**Objective-C**
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+  [ACPSignal registerExtension];
+  // Override point for customization after application launch.
+  return YES;
+}
+```
+
+**Swift**
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  ..............................
+  ACPSignal.registerExtension()
+  ..............................  
+  // Override point for customization after application launch.
+  return true;
+}
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+```jsx
+import {ACPSignal} from '@adobe/react-native-acpcore';
+
+initSDK() {
+    ACPSignal.registerExtension();
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Version of the Signal extension
+
+The `extensionVersion()` API returns the version of the Signal extension that is registered with the MobileCore extension.
+
+To get the version of the Identity extension, use the following code sample:
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+```java
+String signalExtensionVersion = Signal.extensionVersion();
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+
+#### iOS
+
+**Objective-C**
+
+```objectivec
+NSString *signalExtensionVersion = [ACPSignal extensionVersion];
+```
+
+**Swift**
+
+```swift
+var signalExtensionVersion  = ACPSignal.extensionVersion()
+```
+
+{% endtab %}
+
+{% tab title="React Native" %}
+
+#### JavaScript
+
+```jsx
+ACPSignal.extensionVersion().then(signalExtensionVersion => console.log("AdobeExperienceSDK: ACPSignal version: " + signalExtensionVersion));
+```
+
+{% endtab %}
+{% endtabs %}
+
+##Define rules in Adobe Experience Platform Launch
+
+Define the rules in AEP Launch by following the steps in the [Signal extension and Rules engine Integaration document](signals-extension-and-rules-engine-integration)
+
+##Implement the SDK APIs to send PII data to external destinations
+
+To send PII data to external destinations, the `PII` action can trigger the Rules engine when certain triggers and traits match. When setting a rule, you can also set the `PII` action for a Signal event. The `collectPii` API can then be used to trigger the rule and send the PII data.
+
+For the information on collectPii and its usage, see collectPii in [AEP Mobile Core API reference Page](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii)
```

Comment:
please reword to:
"For more information about `collectPII` and its usage, see [collectPII](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#collect-pii).